src/Common/src/Interop/Windows/mincore/Interop.ConsoleScreenBufferInfo.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [StructLayoutAttribute(LayoutKind.Sequential)]
+        internal struct CONSOLE_SCREEN_BUFFER_INFO
```

Comment:
Should this struct and its supporting structs just be in the same file as GetConsoleScreenBufferInfo?  You can't use GetConsoleScreenBufferInfo without them, and we don't expect there to be any other uses of this struct, do we?